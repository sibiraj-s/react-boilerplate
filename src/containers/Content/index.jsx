import ReactLogo from 'Assets/react.svg';

const Content = () => {
  return (
    <div className='flex h-full min-h-screen flex-col justify-center px-4 text-black dark:text-white sm:items-center'>
      <div className='mb-3'>
        <div className='sm:text-center'>
          <img src={ReactLogo} width='250' className='sm:mx-auto' />
          <p className='text-4xl font-medium'>React.js + Webpack boilerplate</p>
        </div>
        <p className='sm:text-center text-sm mt-1 text-gray-600 dark:text-gray-300'>React is a JavaScript library for building user interfaces.</p>
      </div>
      <div className='flex flex-col sm:flex-row mt-4'>
        <a
          className='mb-1 sm:mb-0 sm:mr-1'
          href='https://github.com/sibiraj-s'
          target='_blank'
          rel='noopener noreferrer'
        >
          <img src='https://badgen.net/badge/github/sibiraj-s?icon&color=red' alt='Sibiraj @ Github' />
        </a>
        <a
          className='mb-1 sm:mb-0 sm:mr-1'
          href='https://github.com/sibiraj-s/react-boilerplate'
          target='_blank'
          rel='noopener noreferrer'
        >
          <img src='https://badgen.net/badge/github/react-boilerplate?icon&label' alt='react-boilerplate' />
        </a>
        <a
          href='https://github.com/sibiraj-s/react-boilerplate/blob/master/LICENSE'
          target='_blank'
          rel='noopener noreferrer'
        >
          <img src='https://badgen.net/github/license/sibiraj-s/react-boilerplate?color=green' alt='MIT' />
        </a>
      </div>
    </div>
  );
};

export default Content;
