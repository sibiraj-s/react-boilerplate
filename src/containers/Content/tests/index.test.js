import React from 'react';
import { render } from '@testing-library/react';

import Content from '../index';

describe('<Content />', () => {
  it('should render and match the snapshot', () => {
    const { asFragment } = render(<Content />);
    const renderedOutput = asFragment();
    expect(renderedOutput).toMatchSnapshot();
  });
});
