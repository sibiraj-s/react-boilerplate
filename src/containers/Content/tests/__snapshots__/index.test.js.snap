// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render and match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="h-full min-h-screen flex flex-col justify-center sm:items-center px-4 text-black dark:text-white"
  >
    <div
      class="mb-3"
    >
      <div
        class="sm:text-center"
      >
        <img
          class="sm:mx-auto"
          src="test-file-stub"
          width="250"
        />
        <p
          class="text-4xl font-medium"
        >
          React.js + Webpack boilerplate
        </p>
      </div>
      <p
        class="sm:text-center"
      >
        React is a JavaScript library for building user interfaces.
      </p>
    </div>
    <div
      class="flex flex-col sm:flex-row"
    >
      <a
        class="mb-1 sm:mb-0 sm:mr-1"
        href="https://github.com/sibiraj-s"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Sibiraj @ Github"
          src="https://badgen.net/badge/github/sibiraj-s?icon&color=red"
        />
      </a>
      <a
        class="mb-1 sm:mb-0 sm:mr-1"
        href="https://github.com/sibiraj-s/react-boilerplate"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="react-boilerplate"
          src="https://badgen.net/badge/github/react-boilerplate?icon&label"
        />
      </a>
      <a
        href="https://github.com/sibiraj-s/react-boilerplate/blob/master/LICENSE"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="MIT"
          src="https://badgen.net/github/license/sibiraj-s/react-boilerplate?color=green"
        />
      </a>
    </div>
  </div>
</DocumentFragment>
`;
